package.json

```diff
@@ -200,6 +201,7 @@
     "prettier-plugin-import-sort": "^0.0.7",
     "prettier-plugin-tailwindcss": "^0.1.7",
     "raw-loader": "^4.0.2",
+    "react-test-renderer": "^18.2.0",
```

Comment:
Please revert changes to `package.json` and `yarn.lock` 😄  I think taking on a new testing library dependency is something we would want to discuss and consider on its own, rather than as a side effect of a change like this.